Libraries/builtin/Sources/copy/Driver.cpp

```diff
@@ -41,42 +42,45 @@ name()
 static bool
 CopyPath(Filesystem *filesystem, std::string const &inputPath, std::string const &outputPath)
 {
-    if (!filesystem->createDirectory(FSUtil::GetDirectoryName(outputPath), true)) {
-        return false;
-    }
+    /* Must be writable once copied to allow subsequent builds to overwrite. */
+    Permissions permissions;
+    permissions.user(Permissions::Permission::Write, true);
+    Permissions::Operation operation = Permissions::Operation::Add;
+
+    if (filesystem->isDirectory(inputPath)) {
+        if (!filesystem->copyDirectory(inputPath, outputPath, true)) {
```

Comment:
return filesystem->copyDirectory() && filesystem->writeDirectoryPermissions()?

Same nit for symbolic links and files.